scss/civicrm/common/_tabs.scss

```diff
@@ -1,40 +1,82 @@
 .ui-tabs {
-  background: #ffffff;
+  background: $crm-white;
   border-radius: 0;
   border: 0;
   padding: 0;
+  font-family: $font-family-base;
 
   .ui-tabs-nav {
-    background: #f4f7f8;
+    background: $gray-lighter;
+    border-bottom: 1px solid $crm-dark-grayblue;
     border-radius: 0;
     padding: 10px 10px 0 10px;
 
-    .ui-corner-all {
-      background: #f4f7f8;
-      border-radius: 0;
-      border: 1px solid transparent;
-      padding: 0;
+    *:focus {
+      outline: none;
+    }
+
+    .ui-corner-all,
+    .ui-corner-top {
+      background: $gray-lighter;
+      border:  1px solid transparent;
+      border-bottom: 1px solid $crm-dark-grayblue;
+      border-radius: $border-radius-base $border-radius-base 0 0;
+      font-size: $font-size-base;
+      font-family: $font-family-base;
+      padding: 7px 15px;
+      margin-bottom: -1px;
+
+      &.crm-tab-button {
+        top: 0;
+      }
+
+      span.crm-i {
```

Comment:
do you need to specify `span`? If possible never specify the element in the selector

Reply:
Yes, this one is necessary evil